src/app/CMakeLists.txt

```diff
@@ -116,31 +116,52 @@ if (APPLE)
             MACOSX_PACKAGE_LOCATION Resources)
     set(QT_TR_DIR "${qBittorrent_SOURCE_DIR}/dist/qt-translations")
     set(QT_TRANSLATIONS
+        ${QT_TR_DIR}/qtbase_ar.qm
```

Comment:
AFAIK, we don't need both qt_\*.qm and qtbase_\*.qm for the same language.

Reply:
I added them because I saw them are included in "macxconf.pri", note this is a section for macos.

Reply:
Also there seems to be some differences for qt_\*.qm and qtbase_\*.qm.

https://code.qt.io/cgit/qt/qttranslations.git/tree/translations/qt_zh_TW.ts
This one reference to other files.

https://code.qt.io/cgit/qt/qttranslations.git/tree/translations/qt_zh_CN.ts
This one contains the translation strings.

I think the easiest way is to include both qt_\*.qm and qtbase_\*.qm files so we don't miss out anything.

Reply:
>I think the easiest way is to include both qt_\*.qm and qtbase_\*.qm files so we don't miss out anything.

It's meaningless to include qt_\*.qm if qtbase_\*.qm exists. qBittorrent search for qtbase_\*.qm first and then for qt_\*.qm as a fallback:
https://github.com/qbittorrent/qBittorrent/blob/92cc2a4f31ccf95e412fd8b25e540eeb6f281cea/src/app/application.cpp#L651

Reply:
>It's meaningless to include qt_*.qm if qtbase_*.qm exists. qBittorrent search for qtbase_*.qm first and then for qt_*.qm as a fallback:

OK, I see.
But the way you presented is hard for updating it again in the future. If we don't object having both files included, I can just build the .qm files and copy them without much filtering. Otherwise I have to compare file sizes between qt_\*.qm and qtbase_\*.qm to do the filtering. Unless someone wrote such a tool for this task to lessen the burden, I guess nobody will be interested update it again in the future... so do you insist on this?

Reply:
>Otherwise I have to compare file sizes between qt_*.qm and qtbase_*.qm to do the filtering. 

Why do you need it? If qtbase_\*.qm exists we don't need corresponding qt_\*.qm.

Reply:
You think doing this update is trivial? I'm talking about easing the maintain/update burden.

Reply:
Ok, I've pushed an update that addressed your concern. At this point I don't care about lessen the update burden anymore, I'm going to do this update only once and let others face this non-trivial process in the future.

Reply:
It can't be trivial in General until all Qt translations are separated according to Qt modules and we can use QtBase translation for any language unconditionally.
Currently we have to maintain it manually (or better, in automated mode, if someone implements a helper tool).
Let's consider some language Qt provide all-in-one translation file only for so we distribute qt_\*.qm for it and have appropriate entries in our project files. When the translation becomes "splitted" for this language, we will no longer be able to distribute it using only this file, we will still have to distribute qtbase_\*.qm (add this file and update project files).